src/AiidaDFTK.jl

```diff
@@ -162,20 +192,22 @@ function run_json(filename::AbstractString; extra_output_files=String[])
     end
     push!(all_output_files, timingfile)
 
+    report_error("Finished successfully.")
+
     (; output_files=all_output_files)
 end
 
 
 """
 Run a DFTK calculation from a json input file. The input file name is expected to be passed
-as the first argument when calling Julia (i.e. it should be available via `ARGS`. This
+as the first argument when calling Julia (i.e. it should be available via `ARGS`). This
 function is expected to be called from queuing system jobscripts, for example:
 
 ```bash
 julia --project -e 'using AiidaDFTK; AiidaDFTK.run()' /path/to/input/file.json
 ```
 """
-function run()
+function run(; kwargs...)
```

Comment:
don't do this. `run()` conventionally has no arguments in julia and gets it's info from elsewhere (e.g. environment variables or positional arguments)

Reply:
In fact, I changed it so that the input file is passed as an argument to run. This allows the test to capture the logs easily. I can reintroduce a simple bouncer, but it seems pointless to me. The main advantage of passing the file as a program argument is that one does not need to worry about escaping. However, this is already not a problem in AiiDA since I am not planning to make the input file configurable.

Reply:
Ok, I see. Well fair enough. But what is then even the point between `run()` and `run_json()` ?

Reply:
They are now called `run` and `_run`. I kept the latter to avoid too much nesting in `run`.